plugins/com.redhat.fabric8analytics.lsp.eclipse.ui/.classpath

```diff
@@ -3,7 +3,5 @@
 	<classpathentry exported="true" kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
 	<classpathentry exported="true" kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
 	<classpathentry kind="src" path="src/"/>
-	<classpathentry exported="true" kind="lib" path="library" sourcepath="library"/>
-	<classpathentry exported="true" kind="lib" path="templates" sourcepath="templates"/>
```

Comment:
Local Index.html from templates if referred in the plugin which is not available in the package unless templates folder is added to class paths, is it a good idea to remove this from here?